hyppo/kgof/density.py

```diff
@@ -0,0 +1,682 @@
+"""
```

Comment:
My test statistic calculation and the module as a whole is pretty dependent on the structure of this UnnormalizedDensity object (as well as Data and Kernel) given I needed to easily be able to pull datasources. There's overlapping functionality with sims that exist in numpy, but is there any way these three files can stay under perhaps a more specific name? It would require pretty extensive rewrites to rip these out and am worried it'll compromise some of the functionality.

Reply:
same comment as above

Reply:
Got it. Will focus on it next semester.